### ISSUE TITLE
Some asks following Company Communicator Deployment

### ISSUE THREAD

**Issue by TRCarnell** [OfficeDev/microsoft-teams-apps-company-communicator#585]
Hi team, I'm wondering if someone can help in relation to the below points:

- When sending a message to all, no notification appears in the Activity Feed in Teams. I would like the message to flag in the activity feed
- When sending out company-wide messages, the button is mandatory. Sometimes it's just a message I need to send - no button/ link is required. How can I lose the button from some messages?
- Issues with formatting text in company communicator. In preview mode, the messages are formatted correctly however when published/ sent out, the messages lose their format e.g. bold headings and spacing between paragraphs are lost
- People are blocking messages from Company Communicator. People are starting to block the box and this number goes up every time a message is sent. Is it possible to enforce no blocks?

 Please help or point me in the right direction. Thanks

**Comment by Porkai-Pandian** [comment]
Hi @TRCarnell

Thank you for raising the query, I've provided the response below,

1) It'll not display in activity feed as it is not notification, it'll be considered as new chat, users should see the chat count as 1 if message sent to users
2) You can ignore providing any text to button title and URL, it's optional, however if you enter button title, then button URL is mandatory
3) Yes, this is known [issue](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/issues/567)
4) Not sure if there is a way to enforce to unblock, but would suggest going through [this](https://docs.microsoft.com/en-us/microsoftteams/manage-teams-with-policies) to know more policies set up on Teams which can be performed by Admins

**Comment by Porkai-Pandian** [comment]
Closing this issue due to inactivity, please create new issue in case of any other queries.
